crates/languages/src/typescript/config.toml

```diff
@@ -1,6 +1,6 @@
 name = "TypeScript"
 grammar = "typescript"
-path_suffixes = ["ts", "cts", "d.cts", "d.mts", "mts"]
+path_suffixes = ["ts", "cts", "mts"]
```

Comment:
This change is not important to this PR – and to be honest, I didn't actually test this – but I noticed while working on this that these suffixes a. would never work (because of the issue this PR is trying to fix) and b. are redundant (because `cts` and `mts` are already handled).